### PR TITLE
upd(prismlauncher-git): `7.2`->`8.0`

### PR DESCRIPTION
**Pull request by Xdavius**
solve https://github.com/pacstall/pacstall-programs/issues/4679

### CODE REVIEW

**Comment by Xdavius** [comment]
libqt6core5compat6-dev is for ubuntu 22.04
qt6-5compat-dev for all others

@Elsie19 : Can I set incompatible=(ubuntu:22.04) ?

I have tested with | seems not working

**Comment by Xdavius** [comment]
> Can you conditionally check $DISTRO and use the correct names from there?

go reading the documentation =) and installing ubuntu

File: packages/prismlauncher-git/prismlauncher-git.pacscript [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
 pkgdesc="Custom launcher for Minecraft"
 maintainer="Oren Klopfer <oren@taumoda.com>"
 pkgver() {
   git ls-remote "${url}" develop | cut -f1 | cut -c1-8
 }
 
 prepare() {
+## Provide dependencies changed from VERSION_CODENAME
+source /etc/os-release
+case $VERSION_CODENAME in
```
Comment (Elsie19) [inline packages/prismlauncher-git/prismlauncher-git.pacscript:15]:
```suggestion
case "${DISTRO}" in
```

File: packages/prismlauncher-git/prismlauncher-git.pacscript [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+jammy)
+sudo apt install -y libqt6core5compat6-dev
```
Comment (Elsie19) [inline packages/prismlauncher-git/prismlauncher-git.pacscript:17]:
```suggestion
*:jammy)
  makedepends+=("libqt6core5compat6-dev")
```

File: packages/prismlauncher-git/prismlauncher-git.pacscript [hunk 3]
```diff
@@ -1,15 +1,26 @@
...
+;;
+*)
+sudo apt install -y qt6-5compat-dev
```
Comment (Elsie19) [inline packages/prismlauncher-git/prismlauncher-git.pacscript:20]:
```suggestion
  makedepends+=("qt6-5compat-dev")
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Move these all out of prepare().

**Comment by Elsie19** [comment]
> I had trouble understanding all.
> 
> The documentation https://github.com/pacstall/pacstall/wiki/Pacscript-101 say : 
> 
> ```
> prepare() {
>   ./autogen.sh
>   case "${CARCH}" in
>     amd64) ;;
>     arm64) ;;
>     *) ;;
>   esac
>   case "${DISTRO}" in
>     ubuntu:*) ;;
>     debian:sid) ;;
>     *) ;;
>   esac
> }
> ```

https://github.com/pacstall/pacstall-programs/blob/master/packages%2Fhyprland%2Fhyprland.pacscript#L7-L11

**Comment by Elsie19** [comment]
Also run pre-commit. You should've done that already, it's listed in our setup instructions...

**Comment by Xdavius** [comment]
> Also run pre-commit. You should've done that already, it's listed in our setup instructions...

I use github with the web interface... git from the terminal is not my friend. 
I have to do some extra efforts to learn it. 

Well, this time it's done I hope.

**Comment by Elsie19** [comment]
It's required that you run pre-commit.

**Comment by Xdavius** [comment]
> It's required that you run pre-commit.

Ok. Understand what you are speaking about.

It's the tool that check the code when I do a PR (and use shellcheck and other things)

It has passed successfully.

So I don't understand why the label had changed to invalid here...

**Comment by Elsie19** [comment]
Ok run `shfmt -bn -ci -sr -s -i 2 -w packages/prismlauncher-git/prismlauncher-git.pacscript` and push it then.

**Comment by Xdavius** [comment]
> Ok run `shfmt -bn -ci -sr -s -i 2 -w packages/prismlauncher-git/prismlauncher-git.pacscript` and push it then.

```davius@davius-gladius:~/pacstall-programs$ git config --global user.name "xdavius"
davius@davius-gladius:~/pacstall-programs$ git commit
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...........................................(no files to check)Skipped
check for added large files..............................................Passed
mixed line ending........................................................Passed
shellcheck...............................................................Passed
shfmt....................................................................Passed
update packagelist file..................................................Passed
[master f315fb48] upd(prismlauncher-git): `7.2`->`8.0`
 1 file changed, 11 insertions(+), 2 deletions(-)
davius@davius-gladius:~/pacstall-programs$ git push
remote: Permission to pacstall/pacstall-programs.git denied to Xdavius.
fatal : impossible d'accéder à 'https://github.com/pacstall/pacstall-programs.git/' : The requested URL returned error: 403
```

**Comment by Elsie19** [comment]
`git push origin HEAD`.

**Comment by Xdavius** [comment]
> `git push origin HEAD`.

```
davius@davius-gladius:~/pacstall-programs$ git push origin HEAD
remote: Permission to pacstall/pacstall-programs.git denied to Xdavius.
fatal : impossible d'accéder à 'https://github.com/pacstall/pacstall-programs.git/' : The requested URL returned error: 403
```

**Comment by vigress8** [comment]
1) Create a fork (click on "Fork" on this repo's main page)
2) Clone it
3) If you haven't set up Git with your GitHub credentials, follow these instructions:
https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git#github-cli
4) Create a branch and switch to it
```
git checkout -b <branchname>
```
4) Make your changes and commit them
5) Push
```
 git push -u origin <branchname>
```
6) A branch will be created in your fork, open a PR from there

**Comment by vigress8** [comment]
> I don't understand this command  git push -u origin <branchname>. If I first create a branch, why I need to create an other ?

Because if you created the branch locally, then you need to push it to GitHub now.
-u is the short form of --set-upstream, so you're linking your local branch to a remote branch.

> Is it a way to PR directly from the terminal, or I need to turn to webUI ?
Yes, using `gh` : https://cli.github.com/manual/gh_pr_create

**Comment by Xdavius** [comment]
> Why'd you close this?
I haven't

**Comment by Elsie19** [comment]
image

**Comment by Elsie19** [comment]
Wait there are two PRs..? #5680 and this one.

**Pull request closed without merging**